REQUIRE

```diff
@@ -1,4 +1,4 @@
 julia 0.3
 BinDeps
-Compat
+Compat 0.7.14+
```

Comment:
Should'nt that line be `Compat 0.7.14` ? The Travis failure seems related to that.